src/Microsoft.VisualStudio.Editors/ResourceEditor/ResourceSerializationService.vb

```diff
@@ -95,7 +95,7 @@ Namespace Microsoft.VisualStudio.Editors.ResourceEditor
             Requires.NotNull(Value)
 
             If TypeOf Value IsNot Resource Then
-                Throw CreateArgumentException(NameOf(Value))
+                Return
```

Comment:
It would be good to understand what is going on here, as the method and documentation both expect a `Resource` and only a `Resource`.

Reply:
I can spend some more time looking into it, and then will respond here. I'm _also_ not sure why this is being called.

Reply:
yup I do not understand still

Reply:
I wonder if this was always throwing but then being caught and swallowed further up the stack. Perhaps #9265 changed when this was called such that there was no longer an exception handler in place?

Reply:
That's definitely possible. Since we don't really understand root cause, after discussing with Kay we are just going to revert the initial PR for now